### ISSUE TITLE
Populate the schema_version field in metadata

### ISSUE THREAD

**Issue by maniarathi** [HumanCellAtlas/ingest-central#465]
The metadata schema integration test work will require the `schema_version` to be populated in addition to its current population in `described_by` due to the fact that downstream components will be required to specify metadata versions that they are compatible with and parsing a range based on a URL is not supported. 

This is currently blocked by an effort to establish how to specify the schema_version due to the fact that ElasticSearch is not enabled to search a range of semantic versions (i.e. 3.4.5 -> 5.3.0 is meaningless; only integers or floats are acceptable).

**Comment by daniwelter** [comment]
This issue is a duplicate of https://github.com/HumanCellAtlas/ingest-central/issues/298

**Comment by maniarathi** [comment]
Good catch! Moving information over there.

**Comment by justincc** [comment]
Hi Arathi. Were you intending to implement this? I thought I heard from @morrisonnorman that it's something ingest dev needed to look at, possibly as it's something we used to do but no longer were.

**Comment by maniarathi** [comment]
Hi Justin. There's a duplicate issue that yes, I intend to implement. I doubt it will really need Ingest work though.

But first, the design has to be accepted 😄 which is currently here: https://docs.google.com/document/d/1BInKossmrlpna5MV_s4MSR0Bll1wnD-3OVw-9nCLdWg/edit#

**Comment by justincc** [comment]
Ah oops, didn't see this ticket was closed!

**Comment by justincc** [comment]
I see now. Sorry, wasn't thinking that the work to populate schema_version or major/minor would be further up the stack

**Comment by maniarathi** [comment]
No worries at all! Totally feel free to chime in in the doc as well if you see anything that seems off!
